PiwikTracker/MainThread.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+func assetMainThread(_ file: StaticString = #file, line: UInt = #line) {
+    assert(Thread.isMainThread, "\(file):\(line) must run on the main thread!")
+}
+
+public func asyncMain(_ block: @escaping () -> ()) {
+    DispatchQueue.main.async(execute: block)
+}
```

Comment:
I do like the `assertMainThread` function but I'm not totally convinced by the `asyncMain`. In the end it adds complexity and an extra abstraction for just saving 15 characters. Is it really worth it?

Reply:
I've been so presumptuous to remove this function and replace it by directly calling `DispatchQueue.main.async` instead. I hope you are fine with that. Please let me know if not. I am looking forward to merge this soon.